tests/testthat/test-summary.pop_data.R

```diff
@@ -6,87 +6,33 @@ xs_data <- load_pop_data(
   standardize = TRUE
 )
 
-unstratified_summary <- xs_data %>%
-  summary() %>%
-  magrittr::extract2("age_summary")
-
-test_that("`summary.pop_data()` produces same results when stratified", {
-  min_country <- xs_data %>%
-    summary(strata = "Country") %>%
-    magrittr::extract2("age_summary") %>%
-    pull(age_min) %>%
-    min()
-
-  expect_equal(object = unstratified_summary %>% pull(age_min), expected = min_country)
-})
-
-test_that("`summary.pop_data()` produces same results when stratified", {
-  first_quartile_country <- xs_data %>%
-    summary(strata = "Country") %>%
-    magrittr::extract2("age_summary") %>%
-    pull(age_first_quartile) %>%
-    mean()
-
-  expect_equal(
-    unstratified_summary %>% pull(age_first_quartile) %>% as.numeric(),
-    first_quartile_country,
-    tolerance = 0.5
+test_that("`summary.pop_data()` produces an error when wrong stratification is provided", {
+  expect_error(
+    object = xs_data %>% summary(strata = "province"),
+    regexp = "Element `province` doesn't exist.",
```

Comment:
"Element" should be "Column"

Reply:
It should be "Element" because we are using a regex (`regexp`) to capture the error message (see below) below. It has to be the exact error message thrown otherwise we can't catch the error. 
<img width="504" alt="image" src="https://github.com/UCD-SERG/serocalculator/assets/731279/8ddfdd48-c9c6-4297-964a-42db1cd58652">